### ISSUE TITLE
Show Border of New Block When First Created in Editor

### ISSUE THREAD

**Issue by SwapnilChand** [Cloud-Code-AI/AkiraDocs#180]
**Description:**

When a new block is created in the documentation editor, there is currently no visual feedback indicating its presence, especially when the block is empty. This makes it challenging for users to identify newly created blocks.

**Proposed Solution:**

Implement a border around newly created or empty blocks to provide immediate visual feedback. The border should be hidden once the block contains content, ensuring that users can easily identify new additions without cluttering the interface.

**Steps to Reproduce:**

1. Open the documentation editor.
2. Create a new block (leave it empty).
3. Observe that there is no border or visual indication of the new block's presence.

**Expected Behavior:**

- Upon creating a new or empty block, a border should appear around it, making it easily identifiable.
- Once the block is filled with content, the border should disappear.

**Additional Information:**

This issue has been noted in other contexts, such as the visibility of borders in empty group blocks, which are intended for decorative purposes but lack clear visual feedback in the front-end display. Implementing this feature could enhance user experience significantly by providing clear visual cues during block creation while maintaining a clean interface as content is added.

**Comment by sauravpanda** [comment]
@shreyashkgupta whats your thoughts on this, does it make sense?

**Comment by shreyashkgupta** [comment]
Yeah, borders is one way.  I think the main issue is its very hard to see a new block is added or not.

Another more subtle option could be having a placeholder text. Or having a subtle background color for new blocks.
I think placeholder text might give a more clean look to the editor and still solve the issue

**Comment by SwapnilChand** [comment]
Placeholder is the best option, i agree, and just having the cursor autofocus and blink in this new block should do it.
Though still showing all the placeholders might not be right, I think autofocus + placeholder only when the user focuses on the block would be best.
